packages/cra-template-imodeljs/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "@bentley/cra-template-imodeljs",
+  "version": "0.0.0",
```

Comment:
Let's make the package version 1.0 to start with.

Reply:
Made the version as 1.0.0.